modules/ROOT/nav.adoc

```diff
@@ -120,6 +120,7 @@
 
 
 .xref:release-notes:sdk-release-notes.adoc[Release notes]
+** xref:release-notes:0.7.0-rn.adoc[0.7.1]
```

Comment:
should this be 0.7.1?
```suggestion
** xref:release-notes:0.7.1-rn.adoc[0.7.1]
```

Reply:
@ericswanson-dfinity @lsgunnlsgunn - I don't think so...the change logs I viewed were for 0.7.0...